File: pbweberogwcl/src/app/recuperi/commons/dto/esito-salva-recupero-dto.ts
/*
 * Copyright Regione Piemonte - 2024 
 * SPDX-License-Identifier: EUPL-1.2 
*/

/* tslint:disable */
import {MessaggioDTO} from "../../../shared/commons/dto/messaggio-dto";

export interface EsitoSalvaRecuperoDTO {
  esito?: boolean;
  msgs?: Array<MessaggioDTO>;
}

File: pbservizit/pbservizit-jar/src/main/java/it/csi/pbandi/pbservizit/pbandisrv/dto/recupero/EsitoSalvaRecuperoDTO.java
/*******************************************************************************
* Copyright Regione Piemonte - 2024
* SPDX-License-Identifier: EUPL-1.2
******************************************************************************/

package it.csi.pbandi.pbservizit.pbandisrv.dto.recupero;

public class EsitoSalvaRecuperoDTO implements java.io.Serializable {

	static final long serialVersionUID = 1;
	private Boolean esito;
	private MessaggioDTO[] msgs;
	public Boolean getEsito() {
		return esito;
	}
	public void setEsito(Boolean esito) {
		this.esito = esito;
	}
	public MessaggioDTO[] getMsgs() {
		return msgs;
	}
	public void setMsgs(MessaggioDTO[] msgs) {
		this.msgs = msgs;
	}
	
	

	
	
}
